Formula/crosstool-ng.rb

```diff
@@ -23,12 +24,25 @@ class CrosstoolNg < Formula
   depends_on "grep"
   depends_on "libtool"
   depends_on "m4"
+  depends_on "make"
   depends_on "ncurses" if DevelopmentTools.clang_build_version >= 1000
   depends_on "xz"
 
+  if build.head?
+    depends_on "bash"
```

Comment:
Does it really depend on Bash 4? 😢

Reply:
Yes, https://github.com/crosstool-ng/crosstool-ng/blob/master/bootstrap#L3